test/e2e/volumes.go

```diff
@@ -79,7 +79,7 @@ func startVolumeServer(client *client.Client, config VolumeTestConfig) *api.Pod
 	serverPod := &api.Pod{
 		TypeMeta: api.TypeMeta{
 			Kind:       "Pod",
-			APIVersion: "v1beta3",
+			APIVersion: "v1",
```

Comment:
We should be using latest.Version() at all these places rather than hardcoding "v1beta3" or "v1".